.gitlab-ci.yml

```diff
@@ -181,70 +181,10 @@ re:clang-7:
     - ./tools/ci/travis.sh build CC=clang-7 --enable-debug --enable-Werror --enable-buildbot
     - ./tools/ci/travis.sh test ragnarok ragnarok ragnarok $SQLHOST
 
-pre_re:gcc-4.6:
```

Comment:
remove old compilers if they still supported by hercules probably bad idea.
old debian images if compatible with kernel, can download updates always because debian snapshots. But probably need use special image for this. I remember vicamo or official one swtiched some old images to using snapshots

Reply:
Re-added